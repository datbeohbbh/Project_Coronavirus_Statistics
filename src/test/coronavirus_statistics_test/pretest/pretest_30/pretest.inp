34 23
q 1 10 285
q 1 24 449
q 1 5 121
q 1 20 473
? 4 1
q 1 6 426
? 7
q 1 31 296
q 1 31 382
q 2 31 640
? 7
? 4 3
q 2 31 25
q 1 7 127
? 4 2
? 7
q 1 16 63
? 4 3
? 4 1
? 4 8
? 7
? 4 1
? 4 6