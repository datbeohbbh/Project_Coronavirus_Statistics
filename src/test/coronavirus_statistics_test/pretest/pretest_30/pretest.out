Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 121
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 3 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 285
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 121
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 3 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 121
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 13
-> Has recovered : 665
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 473
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 1 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 13
-> Has recovered : 665
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 426
-> Has recovered : 0
-> Was dead : 0

